File: app/src/main/java/com/mgaetan89/showsrage/activity/ShowActivity.java
package com.mgaetan89.showsrage.activity;

import android.support.v4.app.Fragment;

import com.mgaetan89.showsrage.R;
import com.mgaetan89.showsrage.fragment.ShowFragment;

public class ShowActivity extends BaseActivity {
	@Override
	protected boolean displayHomeAsUp() {
		return true;
	}

	@Override
	protected Fragment getFragment() {
		return new ShowFragment();
	}

	@Override
	protected int getSelectedMenuId() {
		return R.id.menu_shows;
	}

	@Override
	protected int getTitleResourceId() {
		return R.string.show;
	}
}

File: app/src/main/kotlin/com/mgaetan89/showsrage/activity/ShowActivity.kt
package com.mgaetan89.showsrage.activity

import com.mgaetan89.showsrage.R
import com.mgaetan89.showsrage.fragment.ShowFragment

class ShowActivity : BaseActivity() {
    override fun displayHomeAsUp() = true

    override fun getFragment() = ShowFragment()

    override fun getSelectedMenuId() = R.id.menu_shows

    override fun getTitleResourceId() = R.string.show
}
